spotifycli/spotifycli.py

```diff
@@ -253,11 +261,18 @@ def get_spotify_property(spotify_property):
         sys.exit(1)
 
 
-def perform_spotify_action(spotify_command):
-    Popen('dbus-send --print-reply --dest=org.mpris.MediaPlayer2."%s" ' %
-          client +
-          '/org/mpris/MediaPlayer2 org.mpris.MediaPlayer2.Player."%s"' %
-          spotify_command, shell=True, stdout=PIPE)
+def perform_spotify_action(spotify_command, extra_arg = None):
+    command_list = [
+        "dbus-send",
+        "--print-reply",
+        f"--dest=org.mpris.MediaPlayer2.{client}",
+        "/org/mpris/MediaPlayer2",
+        f"org.mpris.MediaPlayer2.Player.{spotify_command}",
+    ]
+    if extra_arg is not None:
+        command_list.append(extra_arg)
+    command_string = " ".join(command_list)  # could avoid this by taking out shell=False below
+    Popen(command_string, shell=True, stdout=PIPE)
```

Comment:
I then put the list back into a string command line and let `Popen` parse it with `shell=True`.  I think it's also reasonable to say `shell=False` and pass in the nice list, but I didn't want to overstep here.  And this works totally fine anyway.